cdap-distributions/src/etc/cdap/conf.dist/cdap-site.xml.example

```diff
@@ -17,119 +17,152 @@
+    </description>
   </property>
-
+  
   <property>
-    <name>hdfs.user</name>
-    <value>yarn</value>
-    <description>User name for accessing HDFS</description>
+    <name>router.bind.port</name>
+    <value>10000</value>
+    <description>
+      Port number that the CDAP router should bind to for HTTP connections
+    </description>
   </property>
 
-  <!--
-    Router configuration
-  -->
-  <!-- Substitue the IP to which Router service should bind to and listen on -->
   <property>
-    <name>router.bind.address</name>
-    <value>LOCAL-ROUTER-IP</value>
-    <description>Specifies the inet address on which the Router service will listen</description>
+    <name>router.server.port</name>
+    <value>${router.bind.port}</value>
+    <description>
+      Router port to which CDAP UI connects
+    </description>
   </property>
 
-  <!--
-    App Fabric configuration
-  -->
-  <!-- Substitute the IP to which App-Fabric service should bind to and listen on -->
+
+  <!-- Applications Configuration -->
+
   <property>
     <name>app.bind.address</name>
-    <value>LOCAL-APP-FABRIC-IP</value>
-    <description>Specifies the inet address on which the app fabric service will listen</description>
+    <value>0.0.0.0</value>
+    <description>
+      Host address on which the app fabric server is started
+    </description>
   </property>
 
-  <!--
-    Data Fabric configuration
-  -->
-  <!-- Substitute the IP to which Data-Fabric tx service should bind to and listen on -->
+
+  <!-- Datasets Configuration -->
+
   <property>
     <name>data.tx.bind.address</name>
-    <value>LOCAL-DATA-FABRIC-IP</value>
-    <description>Specifies the inet address on which the transaction service will listen</description>
+    <value>0.0.0.0</value>
+    <description>
+      Transaction service inet address
+    </description>
   </property>
 
-  <!-- 
-    Kafka Configuration
-  -->
+
+  <!-- Kafka Server Configuration -->
+
+  <property>
+    <name>kafka.default.replication.factor</name>
+    <value>1</value>
+    <description>
+      Kafka replication factor; used to replicate Kafka messages across
+      multiple machines to prevent data loss in the event of a hardware
+      failure. The recommended setting is to run at least two Kafka servers.
+      If you are running two Kafka servers, set this value to 2; otherwise,
+      set it to the number of Kafka servers.
+    </description>
+  </property>
+  
   <property>
     <name>kafka.log.dir</name>
-    <value>/data/cdap/kafka-logs</value>
-    <description>Directory to store Kafka logs</description>
+    <value>/tmp/kafka-logs</value>
+    <description>
+      Kafka log storage directory
+    </description>
   </property>
 
-  <!-- Substitute with a list of all machines which will run the Kafka component -->
   <property>
     <name>kafka.seed.brokers</name>
-    <value>FQDN1:9092,FQDN2:9092</value>
-    <description>List of Kafka brokers (comma separated)</description>
+    <value>127.0.0.1:9092</value>
```

Comment:
While ``127.0.0.1`` is the default value, it will not work on any multi-node clusters.  I think it's better to leave the ``FQDN1:9092,FQDN2:9092`` to force the new user to substitute in hostnames.  Otherwise there will be a lot of misconfigured kafka's out there.